### PR TITLE
added .editorconfig

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true;
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
```
Comment (jhermann) [inline .editorconfig]: Hi, it is somewhat comical you have no newline here. Please fix. :smiley:

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+root = true;
```
Comment (jhermann) [inline .editorconfig]: Please add a comment linking to http://editorconfig.org/ so people know what it is

**Comment by KOLANICH** [comment]
@jhermann

**Comment by jhermann** [comment]
@KOLANICH please read the review comments, then it's a merge

**Comment by KOLANICH** [comment]
@jhermann , I have already fixed that.

**Comment by jhermann** [comment]
Seems you have squashed that change, I simply went by the # of commits.

**Pull request merged**